recipes/qt-msbuild/all/conanfile.py

```diff
@@ -0,0 +1,39 @@
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+required_conan_version = ">=1.53.0"
+
+class QtMSBuildConan(ConanFile):
+    name = "qt-msbuild"
+    description = "Qt/MSBuild MSBuild rules and targets from Qt VS Tools"
+    license = "LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0"
```

Comment:
You can find this license string in any of the packaged files, e.g.: https://github.com/qt-labs/vstools/blob/v3.3.1/QtMSBuild/QtMsBuild/Qt.props#L5